sims/vcs/Makefile

```diff
@@ -36,22 +36,97 @@ debug: $(sim_debug)
 #########################################################################################
 include $(base_dir)/common.mk
 
+#########################################################################################
+# verilator-specific user-interface variables and commands
+#########################################################################################
+HELP_COMPILATION_VARIABLES +=
+HELP_COMMANDS += \
+"   default                = compiles non-debug simulator [./$(shell basename $(sim))]" \
+"   debug                  = compiles debug simulator [./$(shell basename $(sim_debug))]" \
+"   clean                  = remove all debug/non-debug simulators and intermediate files" \
+"   clean-sim              = removes non-debug simulator and verilator-generated files" \
+"   clean-sim-debug        = removes debug simulator and verilator-generated files"
+
 #########################################################################################
 # vcs binary and arguments
 #########################################################################################
 VCS = vcs -full64
 
-VCS_OPTS = -notice -line $(VCS_CC_OPTS) $(VCS_NONCC_OPTS) $(VCS_DEFINE_OPTS) $(EXTRA_SIM_SOURCES)
+PREPROC_DEFINES = \
+	+define+VCS \
+	+define+CLOCK_PERIOD=1.0 \
```

Comment:
VCS args were pulled into `chipyard/vcs.mk` since they are reused in hammer-sim. @colinschmidt

Reply:
This is the main reason why I marked this as "DO NOT MERGE". I have merged it now, but want to test VCS changes on the Millennium machines (might also make some Hammer changes).